deepmd/pt/model/descriptor/sezm_nn/indexing.py

```
@@ -0,0 +1,421 @@
+    device
+        Device for the returned tensor.
+
+    Returns
+    -------
+    torch.Tensor
+        Integer tensor with shape (D,), where D=(lmax+1)^2.
+        Each element is the l value for that position.
+    """
+    lmax = int(lmax)
+    counts = torch.tensor(
+        [2 * l + 1 for l in range(lmax + 1)], device=device, dtype=torch.long
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)